Services/GEV/WBD/classes/Requests/trait.gevWBDRequest.php

```diff
@@ -268,4 +268,25 @@ protected function checkSzenarios() {
 
 		return $errors;
 	}
+
+	/**
+	 * Translates ILIAS values to WBD Values
+	 *
+	 * @return []|gevWBDError[]
+	 */
+	protected function translate($user_id, $row_id, $crs_id = 0) {
+		$dic_errors = array();
+
+		foreach($this->translate_value as $key => $value) {
```

Comment:
`$this->translate_value` is not defined anywhere, i think. When understand the stuff correctly, you use that array locally anyways. Please pass an array reference to translate or return an array of translated arrays and the `$dic_errors`. I would chose the second one together with the php `list` syntax.